migrate/v0_17/testdata/genesis-v16.json

```diff
@@ -1762,7 +1807,7 @@
                   },
                   {
                     "subspace": "incentive",
-                    "key": "HardDelegatorRewardPeriods",
```

Comment:
oh interesting, when was this changed on mainnet?

Reply:
hmm not sure, the name changed in the incentive module sometime year